app/src/main/java/de/westnordost/streetcomplete/statistics/AnswersCounter.java

```diff
@@ -30,7 +30,7 @@
 	private View unsyncedContainer;
 
 	private boolean isFirstUpdateDone;
-	private boolean isAutosync;
+	public boolean isAutosync;
```

Comment:
Instead please put `isAutosync` from `onStart()` in `MainActivity` into own function and use that.

```java
private Prefs.Autosync isAutosync()
{
  return Prefs.Autosync.valueOf(prefs.getString(Prefs.AUTOSYNC,"ON")) == Prefs.Autosync.ON
}
```